### PR TITLE
testing/ettercap: new apport

### PR DESCRIPTION
**Pull request by kazkansouh**
www.ettercap-project.org
Multipurpose sniffer/interceptor/logger for switched LAN

### CODE REVIEW

**Comment by kazkansouh** [comment]
Followed https://wiki.alpinelinux.org/wiki/Creating_an_Alpine_package to create APKBUILD. I added myself as maintainer, but am not sure if this was correct.

File: testing/ettercap/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+# Maintainer: Karim Kanso <kaz.kanso@gmail.com>
+pkgname=ettercap
+pkgver=0.8.2_git20190428
+_gitrev=5fb604823c6063676ebdf7cac66b7807acb4f7ed
+pkgrel=0
+pkgdesc="Multipurpose sniffer/interceptor/logger for switched LAN"
+url="https://www.ettercap-project.org/"
+arch="all"
+license="GPL-2.0-or-later"
+depends=""
```
Comment (maxice8) [inline testing/ettercap/APKBUILD]: empty variables can be removed for brevity

File: testing/ettercap/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+makedepends="cmake make gcc musl-dev ncurses-dev curl-dev libpcap-dev
+	     libnet-dev pcre-dev flex bison linux-headers geoip-dev
+	     check-dev"
+install=""
+subpackages="$pkgname-doc"
+source="ettercap-$pkgver.zip::https://github.com/Ettercap/ettercap/archive/$_gitrev.zip"
+builddir="$srcdir"/$pkgname-$_gitrev
+
+build() {
+	cd "$builddir"
```
Comment (maxice8) [inline testing/ettercap/APKBUILD]: the cd "$builddir" statements can be removed as it is done implicitly now by `abuild`

File: testing/ettercap/APKBUILD (hunk 1 above)
Comment (kazkansouh) [reply testing/ettercap/APKBUILD]: Removed

File: testing/ettercap/APKBUILD (hunk 2 above)
Comment (kazkansouh) [reply testing/ettercap/APKBUILD]: Thanks, didn't realise this.

**Comment by maxice8** [comment]
Squash everything into a single commit.

**Comment by kazkansouh** [comment]
Its been squashed into 1 commit.

**Pull request closed without merging**